doc/classes/Dictionary.xml

```diff
@@ -192,131 +138,148 @@
-				[b]Note:[/b] Don't erase elements while iterating over the dictionary. You can iterate over the [method keys] array instead.
+				Removes the dictionary entry by key, if it exists. Returns [code]true[/code] if the given [param key] existed in the dictionary, otherwise [code]false[/code].
+				[b]Note:[/b] Do not erase entries while iterating over the dictionary. You can iterate over the [method keys] array instead.
 			</description>
 		</method>
 		<method name="find_key" qualifiers="const">
 			<return type="Variant" />
 			<param index="0" name="value" type="Variant" />
 			<description>
-				Returns the first key whose associated value is equal to [param value], or [code]null[/code] if no such value is found.
-				[b]Note:[/b] [code]null[/code] is also a valid key. If you have it in your [Dictionary], the [method find_key] method can give misleading results.
+				Finds and returns the first key whose associated value is equal to [param value], or [code]null[/code] if it is not found.
+				[b]Note:[/b] [code]null[/code] is also a valid key. If inside the dictionary, [method find_key] may give misleading results.
 			</description>
 		</method>
 		<method name="get" qualifiers="const">
 			<return type="Variant" />
 			<param index="0" name="key" type="Variant" />
 			<param index="1" name="default" type="Variant" default="null" />
 			<description>
-				Returns the current value for the specified key in the [Dictionary]. If the key does not exist, the method returns the value of the optional default argument, or [code]null[/code] if it is omitted.
+				Returns the corresponding value for the given [param key] in the dictionary. If the [param key] does not exist, returns [param default], or [code]null[/code] if the parameter is omitted.
 			</description>
 		</method>
 		<method name="has" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="key" type="Variant" />
 			<description>
-				Returns [code]true[/code] if the dictionary has a given key.
-				[b]Note:[/b] This is equivalent to using the [code]in[/code] operator as follows:
+				Returns [code]true[/code] if the dictionary contains an entry with the given [param key].
 				[codeblocks]
 				[gdscript]
-				# Will evaluate to `true`.
-				if "godot" in {"godot": "engine"}:
-				    pass
+				var my_dict = {
+				    "Godot" : 4,
+				    210 : null,
+				}
+
+				print(my_dict.has("Godot")) # Prints true
+				print(my_dict.has(210))     # Prints true
+				print(my_dict.has(4))       # Prints false
 				[/gdscript]
 				[csharp]
-				// You have to use Contains() here as an alternative to GDScript's `in` operator.
-				if (new Godot.Collections.Dictionary{{"godot", "engine"}}.Contains("godot"))
+				var myDict = new Godot.Collections.Dictionary
 				{
-				    // I am executed.
+				    {"Godot", 4},
+				    {210, null},
 				}
```

Comment:
Missing semicolon (`;`) at the end. Since `Variant` is a struct, you can't use `null`, the equivalent would be a default-initialized Variant, I recommend using the `default` keyword for that.

Also I think the preferred style is to have spaces between the braces but we are probably not very consistent in this throughout the documentation.

```csharp
var myDict = new Godot.Collections.Dictionary
{
    { "Godot", 4 },
    { 210, default },
};
```